app-text/mandoc/licenses/mandoc-1.14.3

```diff
@@ -0,0 +1,55 @@
+

```

Comment:
Oops, there's a carriage-return (0x0d) in the first line. (This reminds me popular OS which I don't use.)
It would be nice to drop that CR. I'm OK with keeping an empty line, as long as it does not contain that 0x0d character :)